scene/animation/tween.h

```diff
@@ -179,6 +179,7 @@ class Tween : public Node {
 	real_t tell() const;
 	real_t get_runtime() const;
 
+	Variant interpolate(Variant p_initial_val, Variant p_final_val, real_t p_weight, TransitionType p_trans_type, EaseType p_ease_type);
```

Comment:
Shouldn't this get default values assigned like the other `interpolate_*` methods?